apps/term-apply/pkg/applicant/manager.go

```diff
@@ -14,26 +14,29 @@ type ApplicantManager struct {
 	mu         *sync.RWMutex    // wraps applicant slice access
 	writeChan  chan []applicant // serializes csv file writes
 	resumes    *resumeWatcher
+	bucket     string
+	csvKey     string
 }
 
-const DATA_FILE_KEY = "/term-apply/data/applicants.csv"
-
-func NewApplicantManager(filename, uploadDir string) (*ApplicantManager, error) {
+func NewApplicantManager(filename, uploadDir, bucket, resumePrefix, csvPrefix string) (*ApplicantManager, error) {
```

Comment:
not a fan of this function sig. smells like a refactor down the line.

Reply:
Ya make itd be worth the passing it the config? But I think its reasonable to leave it as-is